### ISSUE TITLE
2 - Components

### ISSUE THREAD

**Issue by juarezpaf** [IonicBrazil/ionic2-docs#3]
Tradução dos seguintes componentes:
`/docs/v2/components/`
- [x] @carloshpdoc Overview
- [x] @carloshpdoc Action Sheets
- [x] @lvdamaceno Alerts
- [ ] @Lucasktrindade Badges
- [ ] @pedrobarrostech Buttons
- [ ] @pedrobarrostech Cards
- [ ] @jjaderg Checkbox
- [ ] @jjaderg Grid
- [ ] @jjaderg Icons
- [ ] @phellipeandrade Inputs
- [ ] @phellipeandrade Lists
- [ ] @phellipeandrade Menus
- [ ] @carloshpdoc Modals
- [ ] @carloshpdoc Navigation
- [ ] @carloshpdoc Radio
- [ ] @carloshpdoc Searchbar
- [ ] @Lucasktrindade Segment
- [ ] @Lucasktrindade Select
- [ ] Slides
- [ ] Tabs
- [ ] @marcoribeirojr Toggle
- [ ] @marcoribeirojr Toolbar

**Comment by angelobms** [comment]
@juarezpaf posso ficar responsável  por esta issue aqui, completa.

**Comment by gabrielclima** [comment]
@juarezpaf o @angelobms tem uma tradução já sendo feita por ele que está em processo de revisão. Acredito que seria melhor ele terminar o outro antes de assinar esses pra ele.

Primeiro [pull request](https://github.com/IonicBrazil/ionic2-docs/pull/11#issuecomment-187861247) que foi analisado.
Segundo [pull request](https://github.com/IonicBrazil/ionic2-docs/pull/14#event-562501301) que foi analisado.
[Cápitulo](https://github.com/IonicBrazil/ionic2-docs/issues/2) que ele se fez voluntário a traduzir.

**Comment by carloshpdoc** [comment]
Olá @juarezpaf  , 
 poderia ficar responsável  por esta issue completa.

**Comment by gabrielclima** [comment]
Pronto @carloshpdoc.

**Comment by carloshpdoc** [comment]
Valeu @gabrielclima  !!!

**Comment by juarezpaf** [comment]
@carloshpdoc novidades por aqui? :blush:

**Comment by lvdamaceno** [comment]
Precisa de ajuda @carloshpdoc ?

**Comment by juarezpaf** [comment]
@lvdamaceno pode assumir essa sessão, pois realmente faz tempo que o @carloshpdoc não envia um feedback ;)

**Comment by lvdamaceno** [comment]
Vou começar então, @carloshpdoc manda um salve se voltar!

**Comment by carloshpdoc** [comment]
Ops @lvdamaceno e @juarezpaf  , voltei, desculpas ... Estive ocupado.
Olhando as datas aqui, nem percebi que se passou tanto tempo(sry), mas estou na área . 
Então, hoje a noite posso subir o Alerts, ok?Já estava pronta, falta uma revisão que darei hoje a noite.

**Comment by carloshpdoc** [comment]
@juarezpaf  vi aqui que o @lvdamaceno já subiu a parte do Alerts, então vou seguir em frente com o Badges e os seguintes a esse. 

Observei  também que não tem link para  o chapters > 2.3 Alerts no README.md principal

**Comment by anterodev** [comment]
@juarezpaf Posso pegar Alerts e Badges?

**Comment by juarezpaf** [comment]
@anterodev fala meu querido. Sinta-se à vontade para contribuir. Se não tiver o nome de ninguém do lado, você pode adicionar o seu e mandar bala. 🚀

**Comment by pedromonad** [comment]
Posso ficar com Buttons e Cards?

**Comment by ednilsonamaral** [comment]
@pedrobarrostech yes, you can! :smile:

**Comment by jdrgoms** [comment]
Fala galera :bowtie:, como estava livre, coloquei meu nome na lista nos seguintes quesitos: 
`Checkbox, Grid & Icons`

**Comment by pedromonad** [comment]
Finalizei a parte dos Buttons, segue para revisão #51

**Comment by phellipeandrade** [comment]
Fiquei com Inputs, Lists e Menus.

**Comment by carloshpdoc** [comment]
Estou pegando o Modals e Navigation.

**Comment by carloshpdoc** [comment]
Pessoal, vou pegar o Radio e o Searchbar.

**Comment by gabrielclima** [comment]
@carloshpdoc mete bala!

**Comment by Lucasktrindade** [comment]
Pessoal como faço para registrar que estou traduzindo os tópicos, consigo traduzir Alerts e Badges, como faço para registrar lá em cima?

**Comment by ednilsonamaral** [comment]
@Lucasktrindade é só informar via comentário na _issue_ mesmo. Aí nós anotamos lá na lista que você está traduzindo! ;)

**Comment by Lucasktrindade** [comment]
Tava vendo na Documentação e já tem pronto a parte de Alertas, tem que colocar que foi concluída com o user da pessoa que fez lá. Eu vou pegar Segment e Select no lugar dos Alerts

**Comment by gabrielclima** [comment]
Verdade.. o @lvdamaceno tinha feito a tradução do Alerts... Coloquei pra tu, @Lucasktrindade!

**Comment by Lucasktrindade** [comment]
Ai a tradução dos badges já ta em processamento, já tem um pull request feito em 11 de abril pelo @carloshpdoc, se puderem substituir lá em cima e me colocarem em Slides e Tabs para organizar e não gerar confusão.

**Comment by gabrielclima** [comment]
> Ai a tradução dos badges já ta em processamento, já tem um pull request feito em 11 de abril pelo @carloshpdoc, se puderem substituir lá em cima e me colocarem em Slides e Tabs para organizar e não gerar confusão.

Galera! Vamos nos atentar pra isso.. Se estiver fazendo uma tradução e seu nome não estiver no capítulo, favor pedir para colocar.

Temo que evitar esse tipo de problema.

**Comment by marcoribeirojr** [comment]
Podem me adicionar em Toggle e Toolbar lá em cima? 
Notei que não tem ninguém nesses dois tópicos.

**Comment by gabrielclima** [comment]
Pronto, @marcoribeirojr!

**Comment by marcoribeirojr** [comment]
@gabrielclima pode me colocar em em sildes e tabs lá em cima caso não esteja em processamento.
Pode me dizer como andam a aceitação dos PR deste tópico?

**Comment by ednilsonamaral** [comment]
Pessoal, apenas avisando vocês que a partir de agora **não é necessário** linkar no README principal e também no README de cada capítulo. Como a documentação oficial ainda vem sofrendo mudanças, nós, da _staff_ iremos atualizar os capítulos posteriormente. Apenas enviem em seu PR o capítulo traduzido.

**Comment by melodevbr** [comment]
@juarezpaf Posso ficar com Slides e Tabs ?
